exporter/datadogexporter/internal/apm/model.go

```diff
@@ -0,0 +1,175 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ *
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2020 Datadog, Inc.
+ */
+package datadogexporter
+
+import (
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/event"
+	"github.com/DataDog/datadog-agent/pkg/trace/obfuscate"
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	"github.com/DataDog/datadog-agent/pkg/trace/sampler"
+	"github.com/DataDog/datadog-agent/pkg/trace/stats"
+	"github.com/DataDog/datadog-agent/pkg/trace/traceutil"
+)
+
+type (
+	// Span contains span metadata
+	span struct {
+		Service  string             `json:"service"`
+		Name     string             `json:"name"`
+		Resource string             `json:"resource"`
+		TraceID  string             `json:"trace_id"`
+		SpanID   string             `json:"span_id"`
+		ParentID string             `json:"parent_id"`
+		Start    int64              `json:"start"`
+		Duration int64              `json:"duration"`
+		Error    int32              `json:"error"`
+		Meta     map[string]string  `json:"meta"`
+		Metrics  map[string]float64 `json:"metrics"`
+		Type     string             `json:"type"`
+	}
+)
+
+// AddTagsToTracePayloads takes a single string of tags, eg 'a:b,c:d', and applies them
+// to each span in a trace
+func AddTagsToTracePayloads(tracePayloads []*pb.TracePayload, tags string) {
```

Comment:
I just went ahead and removed this along with the tests...i think we're gonna have to write these tests from scratch since we're dealing with a pretty significantly pared down version of the serverless code